packages/docusaurus-plugin-structurizr/src/validate-options.ts

```diff
@@ -61,6 +61,7 @@ const Schema = Joi.object<DocusaurusPluginStructurizrOptions>({
   executor: Joi.string().valid('docker', 'cli', 'auto').default('auto'),
   dockerImage: Joi.string().default('structurizr/cli'),
   additionalStructurizrArgs: Joi.string().default(''),
+  ignoreDSLFilesGlob: Joi.string().default('/**/include.*.dsl'),
```

Comment:
**suggestion**: I think we could make it easier for the developer to add multiple ignore patterns, when the plugin accepts an array of globs instead of only one. It may be possible to construct a glob which does that, but the developer experience is not that nice.

What do you think about naming the option `ignorePatterns`, to address the case-sensitivity and length of the name?

```suggestion
  ignorePatterns: Joi.array().items(Joi.string()).default(['/**/include.*.dsl']),
```